src/main/java/uk/gov/hmcts/divorce/caseworker/event/CaseworkerNoticeOfChange.java

```diff
@@ -155,11 +156,26 @@ private CaseData correctRepresentationDetails(final CaseData data,
             data.getApplicant1().setAddress(beforeData.getApplicant1().getAddress());
             if (YES.equals(data.getNoticeOfChange().getAreTheyRepresented())) {
                 data.getApplicant2().setAddress(beforeData.getApplicant2().getAddress());
+                data.getConditionalOrder().getConditionalOrderApplicant1Questions().setIsSubmitted(NO);
```

Comment:
this chunk of code looks identical to the private method created below (setConditionalOrderDefaultValues). Could you just call that instead?